policy/apps.py

```diff
@@ -21,9 +23,18 @@
     "ACTIVATION_OPTION_CONTRIBUTION": 1,
     "CTIVATION_OPTION_PAYMENT": 2,
     "ACTIVATION_OPTION_READY": 3,
-    "contribution_receipt_length": 5
+    "contribution_receipt_length": 5,
+    "comores_features_enabled": False
 }
 
+CALCULATION_RULES = []
+
+def read_all_calculation_rules():
+    """function to read all calculation rules"""
+    for name, cls in inspect.getmembers(importlib.import_module("calculation_comores.calculation_rule"), inspect.isclass):
```

Comment:
you should not need to do that, you should be able to use https://github.com/openimis/openimis-be-calculation_py/blob/546cb3450472e4ae159c6f4ba3d149c8b7668caa/calculation/services.py#L28

Reply:
@delcroip we took inspiration from the other modules (by sending it the elements). I tried to remove this part but it no longer invokes the service automatically.  Shouldn't we just leave this piece of code?

Reply:
I am checking you code locally, the first issue I see is that your module doesn't have the right name, it should start by calcrule not calculation